main/managementinterface.go

```diff
@@ -1196,11 +1231,10 @@ func managementInterface() {
 
 	var addr string
 	if common.IsRunningAsRoot() {
-		addr = managementAddr
+		addr = fmt.Sprintf(":%d", ManagementAddr)
 	} else {
 		addr = ":8000" // Make sure we can run without root priviledges on different port
```

Comment:
Maybe handle the else part too. I like the option to specify a different port, but I think it should also be respected if not running as root.
This is usually when doing development on a dev PC instead of on the PI directly.